package Exceptions;

public class CustomExcep extends Exception {
	public CustomExcep(String str)
	{
		super(str);
	}
	
	public static void main(String args[])
	{
		try
		{
			throw new CustomExcep("haii");
		}
		catch(Exception e )
		{
			System.out.println("r");
		}
		finally
		{
			System.out.println("inside finally");
		}
	}

}
